### ISSUE TITLE
A11y: Low contrast foreground/background combinations

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/browser#2235]
### Steps To Reproduce

Going through the various screens/dialogs of the extension, there are many instances of foreground/background colour combinations for text that fall below the recommended 4.5:1 minimum contrast ratio https://www.w3.org/TR/WCAG21/#contrast-minimum

I'll do a proper extensive check through all the various views and all the themes, but certainly the default light theme is affected.

Question is: do you want me to go through and tweak the actual theme files? Or are they pulled from upstream somewhere / you want to keep them in sync? Ideally, it would be good to get the themes themselves fixed, but in the worst case, offering specific higher-contrast variations (light and dark) could be a possibility.

Looking over the light theme, I don't think the changes needed would be dramatic (for instance, shifting from a foreground colour of `#777777` to `#6D6D6D` would already mop up a large number of problems and get to 4.5:1 or better in most cases)

For non-text content, the scrollbar is currently hardly visible. Ideally should have a contrast ratio of 3:1 https://www.w3.org/TR/WCAG21/#non-text-contrast

### Expected Result

Foreground/background combinations for text should have a colour contrast ratio of 4.5:1 or better

### Actual Result

There are many instances of foreground/background colour for text that fall below 4.5:1 (in most cases, 4:1 or even lower)

### Screenshots or Videos

![screenshot from settings, showing instances of text with 3.93:1 ratio, and one just below 4.5:1 at 4.48:1](https://user-images.githubusercontent.com/895831/146687836-ec609477-8601-4fca-9661-c11ab00b7eba.png)

![red error popup 4.09:1](https://user-images.githubusercontent.com/895831/146690964-cc01acf6-42c0-471f-b149-685151dfa431.png)

![red 'delete' control at 4.09:1 contrast ratio](https://user-images.githubusercontent.com/895831/146687926-3c10e68d-7de0-4f45-9e25-440c3d8ce959.png)

![search placeholder in header 3.21:1](https://user-images.githubusercontent.com/895831/146690855-046be076-40f0-430f-8c5c-8f63b52948d3.png)

![dark mode hovered text / notes field with contrast 4.17:1](https://user-images.githubusercontent.com/895831/146690782-75e4f676-3a2d-4ec6-a6e3-0367ea0c7b15.png)

![dark mode delete control 3.43:1](https://user-images.githubusercontent.com/895831/146690906-f7b65247-bbd1-48d2-94d9-b0ed72d9eb88.png)

![dark mode error popup 3.52:1](https://user-images.githubusercontent.com/895831/146690937-d9f01759-5f6b-4369-ba09-62b1a6ad9fa8.png)

### Operating System

Windows

### Web Browser

Chrome

### Build Version

1.55.0

**Comment by patrickhlauke** [comment]
This issue can be closed at this point. While there are still colour combinations with low contrast, there's been changes to the themes since that make this a bit out of date. Will do a proper recheck of the current state of the colour combinations soon and a matching suggested PR.
